pandas/core/arrays/datetimes.py

```diff
@@ -821,13 +822,14 @@ def day_name(self, locale=None):
 
         Parameters
         ----------
-        locale : string, default None (English locale)
-            locale determining the language in which to return the day name
+        locale : str
+            Locale determining the language in which to return the day name.
+            Default is `None` (English locale).
 
         Returns
         -------
-        month_names : Index
-            Index of day names
+        Index
+            Index of day names.
 
         .. versionadded:: 0.23.0
```

Comment:
I think the `versionadded` would be more useful / standard after the description, not here at the end. @WillAyd what do you think?